### ISSUE TITLE
If all colonists are in a vehicle caravan, this triggers a "everybody is dead or missing message"

### ISSUE THREAD

**Issue by RolandPlass** [SmashPhil/Vehicles#71]
**Describe the bug**
If all colonists are travelling on the worldmap in a vehicle caravan, this triggers a "everybody is dead or missing message"

**To Reproduce**
1. Start a new colony
2. Create a sailboat in godmode
3. From a vehicle caravan and wait for it to start travelling the worldmap
4. Wait on the worldmap for half a day or so
5.  see Error

**Screenshots**
![grafik](https://user-images.githubusercontent.com/6671808/132124031-0dc51c95-9953-4ede-be38-6070d2071456.png)

**Additional context**
Logs https://gist.github.com/bc3b6f3e6f3f5ae2b5ab79b1c2945c03

**Comment by PKPenguin321** [comment]
Seems to be a duplicate of https://github.com/SmashPhil/Vehicles/issues/56
